egs/wsj/s5/steps/cleanup/internal/align_ctm_ref.py

```diff
@@ -0,0 +1,506 @@
+#! /usr/bin/env python
+
```

Comment:
this needs some documentation, preferably both in a comment and in a "description" when you do the args, but at least one or the other.